### ISSUE TITLE
Bagnon update needed

### ISSUE THREAD

**Issue by Paddyman1** [tullamods/Bagnon#1264]
My issue is that this addon is the most popular bag addon in world of warcraft but it has stopped working due to not being updated since early last year 2020...
Now  as of 20/1/2021 it has stopped working at all so please updat this addon so users can keep enjoying the use of this epic addon.

Sincerly: Paddyman

**Comment by Jaliborc** [comment]
which issue you have? it works for me just fine.

**Comment by wkrasl** [comment]
HI Jaliborc.  

First, I want to compliment you on having provided the absolutely most useful Addon I have ever encountered.  I have been using it for years!  Kudos.

Normally I don't worry when there is a minor delay in Bagnon being updated after a WOW update.  I just keep checking daily for the next update.  However, this time I am concerned that there is some kind of misconnect since you are reporting it works fine while others like me find it no longer works.  

My WOW version is 9.0.2, while CurseForge says Bagnon latest version is for WOW 9.0.1.  In WOW, Bagnon shows as "Out of Date" and all bag activity is disabled if I load it anyway.  

The issue might be that you are running an updated version of Bagnon intended for WOW 9.02 on your computer that has not updated correctly on CurseForge.

**Comment by wkrasl** [comment]
I just deleted my prior comment affirming a problem, since all bag activity was failing when loading out of date addons.  However that problem was a few days ago.  I just reran all my own testing again a few minutes ago and bags are now working normally for me -- especially all the great Bagnon features -- even though Bagnon is still showing out of date on WOW 9.0.2.  

I don't know why my bags work now, but it may be because I previously had several *other* out of date addons loaded the same time as the out of date bagnon.  So all of the bag problems were probably caused by a different addon.  Earlier it was acting like all loot was being discarded, so it was probably an out of date junk addon.

In the week since then, I've changed my addon selections, with my current choices shown in the image below.  Many of these are still out of date, but I now see no further problem so I must have replaced or updated a different one that was the actual cause.

Kudos to you for providing such a great addon!  I look forward to updating to your next WOW 9.0.2 update, which right now does not appear to be needed.

<img width="976" alt="Screen Shot 2021-01-23 at 1 00 19 PM" src="https://user-images.githubusercontent.com/17034195/105610421-7c249700-5d7d-11eb-9a87-ec9d5061ea8c.png">
